apis/telemetry/v1alpha1/metricpipeline_types.go

```diff
@@ -63,36 +63,87 @@ type MetricPipelineSpec struct {
 
 // MetricPipelineInput defines the input configuration section.
 type MetricPipelineInput struct {
-	// Configures application related scraping.
-	Application MetricPipelineApplicationInput `json:"application,omitempty"`
-}
-
-// MetricPipelineApplicationInput defines the application input configuration section.
-type MetricPipelineApplicationInput struct {
 	// Configures Prometheus scraping.
 	Prometheus MetricPipelinePrometheusInput `json:"prometheus,omitempty"`
 	// Configures runtime scraping.
 	Runtime MetricPipelineContainerRuntimeInput `json:"runtime,omitempty"`
 	// Configures istio-proxy metrics scraping.
 	Istio MetricPipelineIstioInput `json:"istio,omitempty"`
+	// Configures the collection of push-based metrics which are using the OpenTelemetry protocol.
+	Otlp MetricPipelineOtlpInput `json:"otlp,omitempty"`
 }
 
 // MetricPipelinePrometheusInput defines the Prometheus scraping section.
 type MetricPipelinePrometheusInput struct {
-	// If enabled, Pods marked with `prometheus.io/scrape=true` annotation will be scraped.
+	// If enabled, Pods marked with `prometheus.io/scrape=true` annotation will be scraped. The default is `false`.
```

Comment:
```suggestion
	// If enabled, Pods marked with `prometheus.io/scrape=true` annotation are scraped. The default is `false`.
```

We [avoid future tense](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#tenses).